stdpopsim/catalog/HomSap/__init__.py

```diff
@@ -202,6 +202,37 @@
     )
     _species.add_genetic_map(_gm)
 
+###########################################################
+#
+# Annotations
+#
+#
+#
+###########################################################
+
+_an = stdpopsim.Annotation(
+    species=_species,
+    id="Ensembl_GRCh37_gff3",
+    description="Ensembl GFF3 annotations on GRCh37",
+    long_description="""
+        These are the complete GFF3 annotations from
+        Ensembl. Please see
+        ftp://ftp.ensembl.org/pub/grch37/current/gff3/homo_sapiens/README
+        """,
+    url=(
+        "http://ftp.ensembl.org/pub/grch37/current/gff3/"
+        "homo_sapiens/Homo_sapiens.GRCh37.87.gff3.gz"),
```

Comment:
HomSap is now on GRCh38, so these annotations won't match the genome assembly in the catalog! Is there a way to make the version automagically match up?

Reply:
yup, this will be super easy. all one does is 

1. update the url string, 
2. rerun the maintenance code
3. upload to AWS

however the user will not be aware of an update if they have cached the annotations...